nikola/plugins/compile/rest/chart.py

```diff
@@ -73,52 +73,68 @@ class Chart(Directive):
     has_content = True
     required_arguments = 1
     option_spec = {
-        "copy": directives.unchanged,
+        "box_mode": directives.unchanged,
+        "classes": directives.unchanged,
         "css": directives.unchanged,
+        # "defs": directives.unchanged,
```

Comment:
I thought not to include this option because it should receive a [multiline value](http://www.pygal.org/en/stable/documentation/configuration/rendering.html#defs) and it is kind of hard to write in a rst directive. But it could be accomplished using '\n' and spaces. So it can be included after all. Should I provide another pull request?